package.json

```diff
@@ -1,60 +1,55 @@
 {
   "name": "gb-devtools",
   "version": "0.1.0",
-  "private": true,
```

Comment:
I believe this prevents it from accidentally getting published to npm so we may want to restore that.

Reply:
Done